lib/elixir/lib/calendar.ex

```diff
@@ -1002,8 +1002,11 @@ defmodule Time do
 
   @doc """
   Converts the Time struct to a different calendar.
+
+  Returns `{:ok, %Time{}}` if the conversion was successful,
+  or `{:error, reason}` if it was not, for some reason.
   """
-  @spec convert(Time.t, Calendar.calendar) :: Time.t
+  @spec convert(Time.t, Calendar.calendar) :: {:ok, Calendar.time} | {:error, any}
```

Comment:
They are not but that's the signature you use in other cases, so we should be consistent. We can extend it later if necessary. :)